app/admin.py

```diff
@@ -5,14 +5,30 @@
 
 @admin.register(Client)
 class ClientAdmin(admin.ModelAdmin):
-    pass
+    fields = ('name', 'email', 'phone_number', 'address', 'registration_date',)
+    readonly_fields = ('registration_date',)
+    list_display = ('id', 'name', 'email', 'phone_number', 'registration_date',)
+    list_filter = ('registration_date',)
+    search_fields = ('name', 'email', 'phone_number',)
+    ordering = ('name',)
```

Comment:
Кортеж определяется запятой, а не скобкой.
```suggestion
    fields = 'name', 'email', 'phone_number', 'address', 'registration_date',
    readonly_fields = 'registration_date',
    list_display = 'id', 'name', 'email', 'phone_number', 'registration_date',
    list_filter = 'registration_date',
    search_fields = 'name', 'email', 'phone_number',
    ordering = 'name',
```